MilitaryManager/client-app/src/app/components/profile/profile/profile.component.ts

```diff
@@ -17,7 +18,10 @@ export class ProfileComponent implements OnInit {
   position: string | null | undefined = '';
   unitModel: UnitModel;
 
-  constructor(private _unitUserService : UnitUserService, private _authService: AuthService, private _unitService: UnitsService) {
+  constructor(private _unitUserService : UnitUserService,
+              private _authService: AuthService,
+              private _unitService: UnitsService,
```

Comment:
No need to inject every service apart, you can inject only ApiService that includes all other services